pandas/core/generic.py

```diff
@@ -4464,9 +4463,8 @@ def _convert(self, datetime=False, numeric=False, timedelta=False,
     # TODO: Remove in 0.18 or 2017, which ever is sooner
     def convert_objects(self, convert_dates=True, convert_numeric=False,
                         convert_timedeltas=True, copy=True):
-        """
-        Deprecated.
-        Attempt to infer better dtype for object columns
+        """.. deprecated:: 0.18.0
```

Comment:
same, can also remove the TODO comment (we keep a separate list of these things)